specifyweb/frontend/js_src/lib/components/WorkBench/GeoLocate.tsx

```diff
@@ -61,7 +61,7 @@ export function WbGeoLocate({
       const changes = [
         ['locality.latitude1', latitude],
         ['locality.longitude1', longitude],
-        ['locality.latlongaccuracy', uncertainty],
+        ['locality.geocoorddetails.maxuncertaintyest', uncertainty],
```

Comment:
@grantfitzsimmons I was under impression that GeoLocate already used `geocoorddetails.maxuncertaintyest`. If it was already using `latlongaccuracy`, then there is no inconsistency with Leaflet. Which one should it be using?

Reply:
When I try to recreate the steps from @grantfitzsimmons video, with the code changes from this PR still present on my local machine, I _do not_ get a value in the Max Uncertainty Est field...

Reply:
Looks like the GeoLocate in forms was using `geoCoordDetails.maxUncertaintyEst` like expected, but the code in the workbench was incorrectly using `latlongaccuracy` (along with GeoMaps). This pull request fixes that

@grantfitzsimmons can you try out GeoMap in forms and WorkBench and GeoLocate in forms and WorkBench and check that the `locality.geocoorddetails.maxuncertaintyest` is used in all cases, rather than `latlongaccuracy`